study.md

```diff
@@ -50,15 +50,15 @@ In your own words, define what the responsibilities of the controller layer are
 in Rails.
 
 ```md
-<!-- your answer here -->
+The controller takes server requests/data and sends them to the model
```

Comment:
The router is the one that decides which controller action to execute based on the URL and verb.